website/src/redux/reducers/index.jsx

```diff
@@ -4,13 +4,14 @@
 
 import { combineReducers } from 'redux';
 
-import positions from './positionsReducer';
+import { updateElementReducer, targetElementReducer } from './ElementReducer';
```

Comment:
these should not be in the same file if they are going to be added as separate reducers. This is fine for now, but when we revamp the reducers/actions for the finalized state tree, we should move targetElementReducer out